packages/cli/configs/atk/oauth/python/README.md

```diff
@@ -0,0 +1,40 @@
+# M365 Agents Toolkit Configuration: Oauth
+
+Use this if you want to enable user authentication in your Teams application.
+
+## How to update scopes
+
+1. In the `aad.manifest.json` file, update the `requiredResourceAccess` list to add the required scopes.
+
+2. In the `infra/botRegistration/azurebot.bicep` file, under the `botServicesMicrosoftGraphConnection` resource, update the `properties.scopes` string to be a comma-delimeted list of the required scopes.
+
+### Example
+
+If you want to add the `People.Read.All` and `User.ReadBasic.All` scopes.
```

Comment:
Maybe link to a graph doc which states what the scopes are? https://learn.microsoft.com/en-us/graph/api/user-get?view=graph-rest-1.0&tabs=http User.Read is required here for eg.